posts/using-napari-for-cosmx-data/index.qmd

```diff
@@ -0,0 +1,525 @@
+4. Type `pip install ` into the console (_i.e._, with a single space after the word 'install').
+5. Paste the location of the `whl` file
+6. Press enter to execute
+7. You should receive a message in the console that several packages were successfully
+installed including the `napari-cosmx` package. 
+8. **Close and re-start napari**
+
+```{r}
+#| eval: true
+#| echo: false
+#| label: "fig-windows-plugin-command"
+#| fig-cap: "Example showing how to install the `napari-cosmx` file. Your file name and path will look different."
+
+knitr::include_graphics("./figures/windows_plugin_command.png")
+```
+
+
+::::
+
+
+
+
+### MacOS/Unix Install
+
+::: {.callout-note}
+Depending on your browser and security settings you may get warnings when downloading or running some of the links below.
+:::
+
+#### Part 1: Installing Napari
+
+Visit the napari [0.4.17 release page](https://github.com/napari/napari/releases/tag/v0.4.17){target="_blank"}. 
+Scroll all the way to the bottom and expand the section that says "Assets" (@fig-mac-pkg-download). 
+Download the `pkg` file that is appropriate for your operating system (_i.e._, 
+`napari-0.4.17-macOS-x86_64.pkg` for Mac). Open the downloaded file and install 
+via the instructions on screen (you can accept the defaults). When finished, 
+launch napari via the Applications folder. 
+
+> Pro tip: If launching for the first time, the application may take a moment to appear. Avoid launching multiple instances. 
+
+```{r}
+#| eval: true
+#| echo: false
+#| label: "fig-mac-pkg-download"
+#| fig-cap: "Screenshot of napari packages. Blue highlighted package is appropriate for Mac. Other operating systems' packages are also available but untested."
+
+knitr::include_graphics("./figures/mac_install_pkg.png")
+```
+
+
+
+#### Part 2: Installing `napari-cosmx`
+
+1. In napari, open up the `>_` button that is located on the bottom left (see @fig-mac-plugin-command for example). 
+2. Type `pip install ` into the console (_i.e._, with a single space after the word 'install').
+3. Paste the location of the `whl` file
+4. Press enter to execute
+5. You should receive a message in the console that several packages were successfully
+installed including the `napari-cosmx` package. 
+6. **Close and re-start napari**
+
+```{r}
+#| eval: true
+#| echo: false
+#| label: "fig-mac-plugin-command"
+#| fig-cap: "Example showing how to install the `napari-cosmx` file. Your file name and path will look different."
+
+knitr::include_graphics("./figures/mac_plugin_command.png")
+```
+
+:::
+
+
+## Napari-ready slide folder {#sec-napari-ready-files}
+
+The `napari-cosmx` plugin expects a single slide that has been prepared using the
+`stitching` (@sec-stitching) method within the plugin itself. If you do not already
+have a napari-ready slide, you can download a two-FOV example that we have created
+[here (to do: need to link the two-fov file)](){target="_blank"}. This two-FOV dataset
+was derived from the coronal hippocampus and
```

Comment:
derived from the mouse brain public dataset. Or at least note that this is mouse brain.

Reply:
Done: 208857d88eb7bd81da7fd043536a8aad6f51e28b